### PR TITLE
remove dependency on six

### PR DESCRIPTION
**Pull request by a-detiste**

### CODE REVIEW

File: .github/workflows/build-and-test.yml [hunk 1]
```diff
@@ -62,7 +62,7 @@ jobs:
       - name: Mypy testing
         run: |
           # Not an exact mypy version, as we need 0.942 for pypy-3.8 support, but it's not available on 3.5
```
Comment (alexmojaki) [inline .github/workflows/build-and-test.yml:64]: "but it's not available on 3.5" also sounds like something that can be fixed now, not necessarily here

File: setup.cfg [hunk 2]
```diff
@@ -30,7 +30,6 @@ classifiers =
 [options]
 packages = asttokens
 install_requires =
-    six >= 1.12.0
     typing; python_version < "3.5"
```
Comment (alexmojaki) [inline setup.cfg:33]:
```suggestion
```

**Review by alexmojaki — APPROVED** [review]
Great! Is this ready to merge once the tests pass?

File: .github/workflows/build-and-test.yml (hunk 1 above)
Comment (a-detiste) [reply .github/workflows/build-and-test.yml:64]: Yes ... but I'm really good at managing the Debian ecosystem and releases; not so much the native Python ecosystem (pip, tox etc...)

**Comment by a-detiste** [comment]
It is ready.

**Comment by a-detiste** [comment]
For what it's worth, a much dumber patch was temporarily used in Debian: https://sources.debian.org/src/python-asttokens/2.4.1-2/debian/patches/remove-six.patch/

The good things is that the CI suite of all reverse dependencies (the one using asttokens) passed fine.

```
-import six
+class six:
+     PY2 = False
+     PY3 = True
+     string_types = (str,)
```

**Comment by alexmojaki** [comment]
Thank you so much!!!

**Pull request merged**